### DISCUSSION TITLE
Come intercettare un aumento anomalo di un valore

### DISCUSSION THREAD

**Discussion by vespadj** [indomus/forum#1090] [HUB: Home Assistant]
Si abbia un sensore con un andamento più o meno stabile, oppure blandamente ciclico.

Come determinare una variazione brusca?

Stavo pensando di creare un sensore filtrato da utilizzare come base di comparazione, vedi art. su sensori ipertrofici
https://indomus.it/guide/ridurre-le-scritture-dati-di-sensori-ipertrofici-su-home-assistant/

Avete idee migliori?

**Comment by KC-inDomus** [comment]
> Si abbia un sensore con un andamento più o meno stabile, oppure blandamente ciclico.
> 
> Come determinare una variazione brusca?
> 
> Stavo pensando di creare un sensore filtrato da utilizzare come base di comparazione, vedi art. su sensori ipertrofici https://indomus.it/guide/ridurre-le-scritture-dati-di-sensori-ipertrofici-su-home-assistant/
> 
> Avete idee migliori?

Usa trend https://www.home-assistant.io/integrations/trend/
